Sources/Extensions/SwiftStdlib/DoubleExtensions.swift

```diff
@@ -25,9 +26,80 @@ public extension Double {
 	public var cgFloat: CGFloat {
 		return CGFloat(self)
 	}
+    
+    /// SwifterSwift: Returns the current double (treated as meters) to kilometers.
+    /// Based on Apple's [The Swift Programming Language (Swift 4.0.3) - Extensions](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Extensions.html#//apple_ref/doc/uid/TP40014097-CH24-ID152)
```

Comment:
I think this is fine to list Apple as a source, but let's not expose them in the documentation

Reply:
Noted. Will insert the link in the property body instead. Thanks